qa/workunits/rbd/cli_migration.sh

```diff
@@ -102,23 +135,32 @@ test_import_native_format() {
     local base_image=$1
     local dest_image=$2
 
-    rbd migration prepare --import-only "rbd/${base_image}@2" ${dest_image}
-    rbd migration abort ${dest_image}
+    # if image is from namespace
+    local namespace=""
+    if [[ "$base_image" == rbd/*/* ]]; then
+        namespace=$(basename "$(dirname "$base_image")")
+    fi
+
+    # Execute only if it's a non-namespace based image
+    if [[ -z "$namespace" ]]; then
```

Comment:
Is this `if` condition really needed?  Can `rbd/${base_image}@2` argument to `rbd migration prepare` command be replaced with just `${base_image}@2` so that this test is performed in all configurations?  It wouldn't change the existing logic at all -- the pool is specified redundantly.